api/v1/api.go

```diff
@@ -204,7 +204,6 @@ func (s *apiV1) RegisterRoutes(e *echo.Echo) {
 	netw.GET("/addrs", s.handleNetAddrs)
 
 	miners := public.Group("/miners")
-	miners.GET("", s.handleAdminGetMiners)
```

Comment:
Shouldn't we leave the v1 as it was? Why are we removing this?

Reply:
good catch - shouldn't do this in case it breaks anything.